tests/strings/strings.json

```diff
@@ -193,7 +195,7 @@
                 "options": {}
             }, 
             {
-                "_PTR_F_C_index": 23, 
+                "_PTR_F_C_index": 25, 
```

Comment:
What are these indices used for?

Will  ``"_PTR_F_C_index"`` always have the same index as `` "_function_index"``?

Reply:
When a function is wrapped, sometimes a 'bufferify' version is created.  This is used internally to map between the Fortran function and which C function it actually calls.